policy/cnspec_policy.proto

```diff
@@ -354,22 +358,25 @@ message Control {
   // UID is only needed on Mquery upload, when the MRN is computed.
   // It is not be persisted.
   string uid = 5;
-  repeated ControlRef checks = 7;
-  repeated ControlRef policies = 8;
-  repeated ControlRef controls = 9;
   string title = 20;
   ControlDocs docs = 21;
   map<string,string> tags = 34;
+  // actions applied to this control, only necessary for overrides
+  cnquery.explorer.Action action = 41;
   bool manual = 50; // indicates that this is a manual control
 }
 
 message FrameworkMap {
   string mrn = 1;
-  repeated cnquery.explorer.ObjectRef referenced_policies = 2;
-  repeated ControlMap controls = 3;
+  string uid = 2;
+  string framework = 20;
```

Comment:
nit:
maybe `framework_owner` here? to indicate better the relationship

Reply:
sure, let's give it a shot. We will want to get more feedback from users anyway before this is final